src/Project.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Builder as _Builder} from './Project/Builder';
+
+export namespace Project {
```

Comment:
I think this file is related with `src/Project/*`, right?
Then `Project` means which one among following options?

- Project = `one-build`
- Project = `one-build` + other functions(like showing profiling json)

Reply:
I'm not sure I can catch your question but `Project` here is about project management, like other IDE does.
I've used this term; as vscode-ext does not provide this feature, but we need to manage something like it.
`Project` here is a management of compilation workflow as of now... but not sure this is enough...

So for `Profiling` I could say `Profile.ts` + `Profile` folder; for your question, `src/NewFeature/`

This file+folder management style is from my draft coding and as I am just a novice extension starter,
I am not sure this style is good or bad...

We may have to refactor someday if this blocks something somehow...

Reply:
> So for Profiling I could say Profile.ts + Profile folder; for your question, src/NewFeature/

It seems that `src/Project` manages `ONE build` related features :)
If so, how about changing the name from `Project` to `ONE`, `Onecc`, or other name?
At first, I misunderstood that `Project` includes all of our features.
But if `src/dir1` manages only `dir1` related features, I think `Project` is so general to distinguish different features :)

Reply:
As I wrote, this structure is may be refactored later.
Currently I do not want to change to `ONE` or `Onecc` as (1) I do not know `ONE` is also appropriate (2) I may messup the draft code that may not work anymore; which will take more time to solve.

Reply:
Are you willing to use term `Project` ?

Reply:
I am not objecting to the structure, just asked about name.
I will not use the term `Project` but reason for question was just because `Project` seems to manage all the "project" in this repository.